modules/framework/galasa-parent/dev.galasa.framework.k8s.controller/src/main/java/dev/galasa/framework/k8s/controller/InterruptedRunEventProcessor.java

```diff
@@ -59,8 +60,22 @@ public void run() {
                 if (interruptEvent == null) {
                     isDone = true;
                 } else {
-                    markRunFinishedInDss(interruptEvent);
-                    processRasActions(interruptEvent);
+                    String runName = interruptEvent.getRunName();
+                    List<RunRasAction> rasActions = interruptEvent.getRasActions();
+                    rasActionProcessor.processRasActions(runName, rasActions);
+                    
+                    String interruptReason = interruptEvent.getInterruptReason();
+                    switch (interruptReason) {
+                        case Result.CANCELLED:
+                        case Result.HUNG:
```

Comment:
What decides that it's hung ?

Reply:
Currently, only the dead heartbeat monitor sets this `Hung` result (see changes in https://github.com/galasa-dev/galasa/pull/276/files#diff-aaf2f845a8022ce883024fcd2a174b2604a61f0cdedf009c742cf0e30702d8b8).